plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TransactionLogParser.java

```diff
@@ -136,7 +137,7 @@ public static DeltaLakeTransactionLogEntry parseJson(String json)
         if (json.endsWith("x")) {
             json = json.substring(0, json.length() - 1);
         }
-        return OBJECT_MAPPER.readValue(json, DeltaLakeTransactionLogEntry.class);
+        return JsonUtils.parseJson(OBJECT_MAPPER, json, DeltaLakeTransactionLogEntry.class);
```

Comment:
nit: static import

Reply:
this is within local `parseJson` method (name conflict)